### PR TITLE
Mean, Geometric Mean & Harmonic mean Functions implemented in Statistics.py module

### PR DESCRIPTION
**Pull request by Madhav2310**

### CODE REVIEW

**Comment by Madhav2310** [comment]
@certik @czgdp1807 @namannimmo10 please review

File: integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -176,6 +176,7 @@ RUN(NAME test_max_min        LABELS cpython llvm)
 RUN(NAME test_integer_bitnot LABELS cpython llvm)
 RUN(NAME test_unary_minus    LABELS cpython llvm)
 RUN(NAME test_unary_plus     LABELS cpython llvm)
+RUN(NAME test_statistics     LABELS cpython)
```
[the remark this answers is not in the document]
Comment (Madhav2310) [reply integration_tests/CMakeLists.txt:179]: llvm tests weren't passing, I think it had something to do with list implementation in the backend

[the remark this answers is not in the document]
Comment (Madhav2310) [reply integration_tests/CMakeLists.txt:179]: Getting the following error: 
```
semantic error: Arguments do not match for any generic procedure, mean
  --> /home/madhav/lpython/integration_tests/test_statistics.py:11:9
   |
11 |     j = mean(b)
   |         ^^^^^^^ 


```

**Review by certik — APPROVED** [review]

**Comment by certik** [comment]
I think that this is fine.

Can you please isolate the bugs that prevent LPython to compile this, and report them as issues? Let's get them fixed.

**Pull request merged**